### PR TITLE
[OPENGLCFG] 640x480-ify

### PR DESCRIPTION
**Pull request by JoachimHenze**
A simple one

## Purpose

en-US before (did not fit 640x480):
en-us_before

en-US after (does fit 640x480):
en-us_after

JIRA issue: none

### CODE REVIEW

**Review by SigmaTel71 — COMMENTED** [review]
Why email was preserved only for ro-RO, but not tr-TR?

File: dll/cpl/openglcfg/lang/tr-TR.rc [hunk 1]
```diff
@@ -1,8 +1,8 @@
-/* TRANSLATOR: 2017, 2020 Erdem Ersoy (eersoy93) (erdemersoy [at] erdemersoy [dot] net) */
+// COPYRIGHT: 2017-2020 Erdem Ersoy
```
Comment (SigmaTel71) [inline dll/cpl/openglcfg/lang/tr-TR.rc:1]:
```suggestion
// COPYRIGHT: 2017-2020 Erdem Ersoy <erdemersoy@erdemersoy.net>
```

**Review by HBelusca — APPROVED** [review]
LGTM but please fix what SigmaTel71 is pointing before merging.

File: dll/cpl/openglcfg/lang/tr-TR.rc (hunk 1 above)
Comment (JoachimHenze) [reply dll/cpl/openglcfg/lang/tr-TR.rc:1]: No, this email address is outdated. Erdem Ersoy changed his email address about 3 or 4 times over the last 4 years (as often as others do change their underwear). And this one is outdated already and not longer active.

**Pull request merged**